### PR TITLE
extensiq recipe

### PR DESCRIPTION
**Pull request by WRKampi**
extensiq recipe, created by grayskull

### CODE REVIEW

File: recipes/extensisq/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/WRKampi/extensisq
+  summary: extend solve_ivp of scipy.integrate with OdeSolver objects
+  license: MIT
+  license_file: LICENSE
```
Comment (synapticarbors) [inline recipes/extensisq/meta.yaml:39]: Add a comment:

```yaml
# License file manually packaged. See: https://github.com/WRKampi/extensisq/pull/1
license_file: LICENSE
...
```

I've opened a PR upstream (https://github.com/WRKampi/extensisq/pull/1) to fix having to manually package the license file. In a future release if this PR is merged or if the license file is included in some other way, this comment is a visual reminder to check, and if the license file is in the source distribution, you can remove the comment and delete the manually packaged copy of the license.

**Review by synapticarbors — COMMENTED** [review]
Minor change requested, but otherwise looks good.

File: recipes/extensisq/meta.yaml (hunk 1 above)
Comment (WRKampi) [reply recipes/extensisq/meta.yaml:39]: I have made the modifications. The packaged file now contains the licence. I created a new version for this. Is this the only way to do it? Or did I make it more difficult that I needed to? It is my first contribution to conda forge.

**Comment by synapticarbors** [comment]
@WRKampi -- thank you for your contribution to conda-forge! If you run into any issues once the feedstock has been created, don't hesitate to reach out for help (see also: https://conda-forge.org/docs/orga/getting-in-touch.html).

**Pull request merged**